pyuvdata/tests/test_mwa_beam.py

```diff
@@ -103,27 +104,45 @@ def test_bad_amps():
 def test_bad_delays():
     beam1 = UVBeam()
 
-    delays = np.zeros([2, 8])
+    delays = np.zeros([2, 8], dtype='int')
     with pytest.raises(ValueError) as cm:
         beam1.read_mwa_beam(filename, pixels_per_deg=1, delays=delays)
     assert str(cm.value).startswith('delays must be shape')
 
-    delays = np.zeros((2, 16))
+    delays = np.zeros((2, 16), dtype='int')
     delays = delays + 64
     with pytest.raises(ValueError) as cm:
         beam1.read_mwa_beam(filename, pixels_per_deg=1, delays=delays)
     assert str(cm.value).startswith('There are delays greater than 32')
 
+    delays = np.zeros((2, 16), dtype='float')
+    with pytest.raises(ValueError) as cm:
+        beam1.read_mwa_beam(filename, pixels_per_deg=1, delays=delays)
+    assert str(cm.value).startswith('Delays must be integers.')
+
 
 def test_dead_dipoles():
     beam1 = UVBeam()
 
-    delays = np.zeros((2, 16))
+    delays = np.zeros((2, 16), dtype='int')
     delays[:, 0] = 32
 
     uvtest.checkWarnings(
         beam1.read_mwa_beam, func_args=[filename],
         func_kwargs={'pixels_per_deg': 1, 'delays': delays},
         message=('There are some terminated dipoles'))
 
-    assert beam1.check()
+    delay_str = ('[[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], '
+                 '[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]]')
+    gain_str = ('[[0.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, '
+                '1.0, 1.0, 1.0, 1.0, 1.0], '
+                '[0.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, '
+                '1.0, 1.0, 1.0, 1.0]]')
+    history_str = ('Sujito et al. full embedded element beam, derived from '
+                   'https://github.com/MWATelescope/mwa_pb/'
+                   + '  delays set to ' + delay_str + '  gains set to ' + gain_str
+                   + beam1.pyuvdata_version_str)
+    print(beam1.history)
```

Comment:
I think we can get rid of these print statements.

Reply:
oops, thanks!